migrations/versions/60_add_acknowledged_column.py

```diff
@@ -0,0 +1,41 @@
+"""empty message
+
+Revision ID: 2d2309ce61f
+Revises: 50_remove_updated_details
+Create Date: 2015-06-17 11:40:51.427138
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '60_add_acknowledged_column'
+down_revision = '100_add_open_status'
```

Comment:
The down revision is `100_add_open_status` but this revision is `60_add_acknowledged_column`. Also, it doesn't match what's in the comment above.